recipes/automake/all/conanfile.py

```diff
@@ -102,15 +102,17 @@ def package(self):
                 os.rename(fullpath, fullpath + ".exe")
 
     def package_info(self):
+        self.cpp_info.libdirs = []
+
         bin_path = os.path.join(self.package_folder, "bin")
-        self.output.info("Appending PATH env var with : {}".format(bin_path))
+        self.output.info("Appending PATH environment variable:: {}".format(bin_path))
         self.env_info.PATH.append(bin_path)
 
         bin_ext = ".exe" if self.settings.os == "Windows" else ""
 
         aclocal = tools.unix_path(os.path.join(self.package_folder, "bin", "aclocal" + bin_ext))
-        self.output.info("Setting ACLOCAL to {}".format(aclocal))
-        self.env_info.ACLOCAL = aclocal
+        self.output.info("Appending ACLOCAL environment variable with: {}".format(aclocal))
+        self.env_info.ACLOCAL.append(aclocal)
```

Comment:
does it make sense to append?

Reply:
Normally, `ACLOCAL` should be used instead of `AUTOMAKE_CONAN_INCLUDES`.